app/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Contact;
+use Auth;
+
+class ContactController extends Controller
+{
+
+    function store(Request $request)
+    {
+        // dd($request);
+        $contact = new Contact();
+
+        $contact -> name = $request -> name;
+        $contact -> mail = $request -> mail;
```

Comment:
contact.blade.php でinputタグのname属性には `email` と入っているのでここも右辺はemailにしてください 👀 
```suggestion
        $contact -> mail = $request -> email;
```